pkg/vm/vm_test.go

```diff
@@ -2713,6 +2713,45 @@ func TestREMOVEMap(t *testing.T) {
 	assert.Equal(t, makeStackItem(false), vm.estack.Pop().value)
 }
 
+func testCLEARITEMS(t *testing.T, items ...StackItem) {
```

Comment:
Why is it variadic?

Reply:
I have first come up with another set of tests.

Reply:
Fixed.